pandas/tests/io/json/test_readlines.py

```diff
@@ -34,6 +34,7 @@ def test_read_jsonl(engine):
 
 
 def test_read_jsonl_engine_pyarrow(json_dir_path, engine):
+    print(os.path.join(json_dir_path, "line_delimited.json"))
```

Comment:
Sorry, this is my mistake, I will remove it in the next update. I was trying to figure out why an xfail test was failing.